docs/development/setup-advanced.md

```diff
@@ -92,6 +92,9 @@ Store.
    and [connecting the Zulip upstream repository][zulip-rtd-git-connect]:
 
    ```
+   # NOTE: Clone zulip inside the WSL disk and not on Windows mounted disk.
+   # You will run into permission issues if you run provision from zulip
+   # in Windows mounted disk.
```

Comment:
Should we write this as a step rather than a NOTE inside the code block?

(Also, I wonder if our "Clone your fork") link needs to change?

Reply:
Converted NOTE into a step. Clone your fork link is correct I checked. Where do you want it to change to?